cmd/neofs-adm/internal/modules/morph/balance.go

```diff
@@ -54,12 +53,13 @@ func dumpBalances(cmd *cobra.Command, _ []string) error {
 	inv := invoker.New(c, nil)
 
 	if dumpStorage || dumpAlphabet || dumpProxy {
-		nnsCs, err = c.GetContractStateByID(1)
```

Comment:
is the juice worth the squeeze? it's almost always the same or even more code lines in the current adm when you need to get hash. also, nns package has public `ID` const

Reply:
It's the same when we need hash. But there is potential to drop even more code, `nnsResolveHash` can be dropped eventually.

Reply:
> It's the same when we need hash

this exact code part is a 3 > 4 change

> But there is potential to drop even more code

i do not doubt using nns wrappers is the best way to go, i just doubt this PR makes it any easier/better/simpler, i would refactor it some deeper or keep it as is

Reply:
I've cut a bit more lines now and we'll do more after https://github.com/nspcc-dev/neofs-contract/pull/397.